BackendApartmentReservation/BackendApartmentReservation/Controllers/ProfilesController.cs

```diff
@@ -43,5 +44,29 @@ public async Task<IActionResult> Register([FromBody] RegisterRequest model)
             return Ok(responseId);
 
         }
+
+        [HttpGet]
+        [Route("profiles/{userId:int}")]
+        public async Task<IActionResult> GetUserById(int userId)
+        {
+            string userIdInString = userId.ToString();
+
+            try
+            {
+                var result = await _userManager.FindByIdAsync(userIdInString);
+                var response = new GetUserResponse()
+                {
+                    FirstName = result.FirstName,
+                    LastName = result.LastName,
+                    Email = result.Email,
+                    Office = result.Office
+                };
+                return Ok(response);
+            }
+            catch (NullReferenceException)
+            {
+                return BadRequest($"User with ID={userId} was not found.");
```

Comment:
Have the same notice as for Andrėjus. In my opiniom, it is better not to return human readable errors for the frontend. What we have done - we were using `ErrorCodes` class which has error codes specifically for the frontend, without any special characters. It could look like `employee.employee_not_found`

Reply:
Updated with new commit.